include/QuantumComputation.hpp

```diff
@@ -746,7 +752,7 @@ class QuantumComputation {
     dump(std::move(of), format);
   }
   virtual void dump(std::ostream&& of, Format format);
-  virtual void dumpOpenQASM(std::ostream& of);
+  virtual void dumpOpenQASM(std::ostream& of, bool openQASM3 = false);
```

Comment:
```suggestion
  virtual void dumpOpenQASM(std::ostream& of, bool openQASM3);
```
clang-tidy complains here because default arguments on virtual methods are prohibited.
How bad (mostly in the sense of code duplication) would it be to just define a `dumpOpenQASM3` method? (if both are the same than one might just call the other after all)